packages/blade/src/components/Dropdown/DropdownHeaderFooter.tsx

```diff
@@ -73,21 +73,6 @@ const _DropdownFooter = ({ children, testID }: DropdownFooter): React.ReactEleme
     <BaseBox
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       ref={footerRef as any}
-      {...(isReactNative()
```

Comment:
What's happening here? Why are we deleting this?

Reply:
One of the piece of code that was taking care of not consider footer actions as outside click when focus was used.

No need of this now